assignment/tool/src/webapp/vm/assignment/chef_assignments_list_assignments.vm

```diff
@@ -491,6 +496,24 @@
 									#end
 								#end
 							</td>
+							#if ($allowAddAssignment == false)
+								<td headers="grade">
+									#if ($service.getSubmissionStatus($submission.Id, false) == $tlang.getString("gen.returned") && $submission.grade)
+										#if ($assignment.TypeOfGrade.ordinal() == 2)
```

Comment:
I feel you're doing a similar operation than 'getGradeDisplay', can you confirm?

Reply:
Yes, i could getting the grade with the "getGradeDisplay" function. Now, i change all